virtool/downloads/db.py

```diff
@@ -64,7 +64,7 @@ async def generate_sequence_fasta(db, sequence_id: str) -> Tuple[str, str]:
     return virtool.downloads.utils.format_fasta_filename(otu_name, isolate_name, sequence["_id"]), fasta
 
 
-async def generate_otu_fasta(db, otu_id: str) -> Tuple[str, str]:
+async def generate_otu_fasta(db, otu_id: str) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
This should probably be moved to `virtool.otus.db`. Is it only used in `virtool.otus`?

I want to eventually drop `virtool.downloads`.

Reply:
Yes, I can do that.